app/views/docs/realtime.phtml

```diff
@@ -405,10 +405,10 @@ subscription.close()</code></pre>
                 timestamp
             </td>
             <td data-title="Type: ">
-                integer
+                string
             </td>
             <td data-title="Description: ">
-                The <a href="https://en.wikipedia.org/wiki/Unix_time" rel="noopener" target="_blank">UNIX timestamp</a> from the server to ensure consistency across all client platforms and real-time technologies.
+                The <a href="https://en.wikipedia.org/wiki/ISO_8601" rel="noopener" target="_blank">ISO 8601</a> in UTC timezone from the server to ensure consistency across all client platforms and real-time technologies.
```

Comment:
> The ISO 8601

This doesn't quite make sense because ISO 8601 is a standard. Maybe:

```suggestion
                The <a href="https://en.wikipedia.org/wiki/ISO_8601" rel="noopener" target="_blank">ISO 8601 timestamp</a> in UTC timezone from the server to ensure consistency across all client platforms and real-time technologies.
```